lib/auth/new_web_user.go

```diff
@@ -40,6 +40,19 @@ import (
 	"github.com/tstranex/u2f"
 )
 
+func (s *AuthServer) getTOTPLocation() string {
```

Comment:
Missing function comment.

Reply:
It should be there for unexported functions too? okay, will add now.
```suggestion
// Returns the server name for authenticator
func (s *AuthServer) getTOTPLocation() string {
```